source/test/F0.Minesweeper.Components.Tests/Logic/Cell/CellStatusManagerTests.cs

```diff
@@ -10,80 +10,151 @@ namespace F0.Minesweeper.Components.Tests.Logic.Cell
-		[InlineData(CellStatusType.Undefined, MouseButtonType.Left)]
-		[InlineData(CellStatusType.Undefined, MouseButtonType.Middle)]
-		[InlineData(CellStatusType.Undefined, MouseButtonType.Right)]
-		public void CanNext_UndefinedTransition_ReturnsFalse(CellStatusType currentStatus, MouseButtonType command)
+		[InlineData(CellStatusType.Covered, MouseButtonType.Middle, null)]
+		[InlineData(CellStatusType.Covered, MouseButtonType.Middle, true)]
+		[InlineData(CellStatusType.Covered, MouseButtonType.Middle, false)]
+		[InlineData(CellStatusType.Covered, MouseButtonType.Right, true)]
+		[InlineData(CellStatusType.Covered, MouseButtonType.Right, false)]
+		[InlineData(CellStatusType.Flagged, MouseButtonType.Left, null)]
+		[InlineData(CellStatusType.Flagged, MouseButtonType.Left, true)]
+		[InlineData(CellStatusType.Flagged, MouseButtonType.Left, false)]
+		[InlineData(CellStatusType.Flagged, MouseButtonType.Middle, null)]
+		[InlineData(CellStatusType.Flagged, MouseButtonType.Middle, true)]
+		[InlineData(CellStatusType.Flagged, MouseButtonType.Middle, false)]
+		[InlineData(CellStatusType.Flagged, MouseButtonType.Right, true)]
+		[InlineData(CellStatusType.Flagged, MouseButtonType.Right, false)]
+		[InlineData(CellStatusType.Unsure, MouseButtonType.Middle, null)]
+		[InlineData(CellStatusType.Unsure, MouseButtonType.Middle, true)]
+		[InlineData(CellStatusType.Unsure, MouseButtonType.Middle, false)]
+		[InlineData(CellStatusType.Unsure, MouseButtonType.Right, true)]
+		[InlineData(CellStatusType.Unsure, MouseButtonType.Right, false)]
+		[InlineData(CellStatusType.Uncovered, MouseButtonType.Left, null)]
+		[InlineData(CellStatusType.Uncovered, MouseButtonType.Left, true)]
+		[InlineData(CellStatusType.Uncovered, MouseButtonType.Left, false)]
+		[InlineData(CellStatusType.Uncovered, MouseButtonType.Middle, null)]
+		[InlineData(CellStatusType.Uncovered, MouseButtonType.Middle, true)]
+		[InlineData(CellStatusType.Uncovered, MouseButtonType.Middle, false)]
+		[InlineData(CellStatusType.Uncovered, MouseButtonType.Right, null)]
+		[InlineData(CellStatusType.Uncovered, MouseButtonType.Right, true)]
+		[InlineData(CellStatusType.Uncovered, MouseButtonType.Right, false)]
+		[InlineData(CellStatusType.Undefined, MouseButtonType.Left, null)]
+		[InlineData(CellStatusType.Undefined, MouseButtonType.Left, true)]
+		[InlineData(CellStatusType.Undefined, MouseButtonType.Left, false)]
+		[InlineData(CellStatusType.Undefined, MouseButtonType.Middle, null)]
+		[InlineData(CellStatusType.Undefined, MouseButtonType.Middle, true)]
+		[InlineData(CellStatusType.Undefined, MouseButtonType.Middle, false)]
+		[InlineData(CellStatusType.Undefined, MouseButtonType.Right, null)]
+		[InlineData(CellStatusType.Undefined, MouseButtonType.Right, true)]
+		[InlineData(CellStatusType.Undefined, MouseButtonType.Right, false)]
+		public void CanNext_UndefinedTransition_ReturnsFalse(CellStatusType currentStatus, MouseButtonType command, bool? isMine)
```

Comment:
Method under test is named `CanMoveNext` , but this and the next test-method just start with `CanNext`.

Reply:
The renaming we once defined definitely got lost on the tests. As I recall the mut was called `CanNext` initially. Changed the test name!